tests/server_test.py

```diff
@@ -23,6 +57,29 @@ def test_myapp_basic(fc):
     pkunit.pkre("elegant.*myapp.*srw", pkcompat.from_bytes(r.data))
 
 
+def test_simulation_redirect(fc):
+    fc.sr_get("simulationRedirect", redirect=False).assert_http_redirect("simulations")
+
+
+def test_simulation_schema(fc):
+    from pykern.pkcollections import PKDict
+    from pykern.pkunit import pkexcept
+
+    r = fc.sr_post_form(
+        "simulationSchema",
+        data=PKDict(simulationType=fc.sr_sim_type),
+    )
+
+    for k in ("model", "view"):
+        assert k in r.keys()
```

Comment:
Maybe check a specific value, e.g.

`pkeq("/simulation-schema", r.route.simulationSchema)`

Testing for keys is probably ok, too, but it's pretty arbitrary, and a one-liner is sufficient for this particular purpose.

Reply:
I'm not really understanding the benefit of that example. Are we not testing to see that it returned a valid schema? I added a check more specific to myapp, let me know what you think. 1a08a18cb5e9c42888b493a7621bd56f9ac7faea